src/map/attack.cpp

```diff
@@ -217,6 +235,11 @@ uint8 CAttack::GetAnimationID()
         return this->m_attackDirection == RIGHTATTACK ? 2 : 3;
     }
 
+    if (this->m_attackType == PHYSICAL_ATTACK_TYPE::DAKEN)
+    {
+        return 4;
```

Comment:
can we enum attack directions

Reply:
nevermind, just read the function name now

still don't like the magic numbers though im not sure what we should use

Reply:
ATTACK_ANIMATIONS to attack.h?